compiler/lit.site.cfg.py

```diff
@@ -0,0 +1,17 @@
+# Autogenerated from /home/jmonsalv/develop/dataflow_dependencies/iree/compiler/lit.site.cfg.py.in
```

Comment:
drop this file? it has a bunch of hard coded paths

I'm not sure we need any of this - if we do, we need to make sure it works on all platforms (mac/windows/linux) for all users - I'd separate this out from this PR and get broader review

Reply:
Shoot I didn’t mean to add this file. Sorry. This is an output.